gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/IncrementGuard.kt

```diff
@@ -34,7 +34,19 @@ class IncrementGuard : Plugin<Project> {
         const val taskName = "checkVersionIncrement"
     }
 
+    /**
+     * Adds the [CheckVersionIncrement] task to the project.
```

Comment:
Is `[CheckVersionIncrement]` a special notation used to mention Gradle tasks? If not, can we just wrap it into backticks instead?

Reply:
It's a `{@link }` in Kotlin.